src/csharp/Grpc.Core/Internal/BatchContextSafeHandle.cs

```diff
@@ -92,7 +92,8 @@ public ClientSideStatus GetReceivedStatusOnClient()
             UIntPtr detailsLength;
             IntPtr detailsPtr = Native.grpcsharp_batch_context_recv_status_on_client_details(this, out detailsLength);
             string details = MarshalUtils.PtrToStringUTF8(detailsPtr, (int)detailsLength.ToUInt32());
-            var status = new Status(Native.grpcsharp_batch_context_recv_status_on_client_status(this), details);
+            string error = Marshal.PtrToStringAnsi(Native.grpcsharp_batch_context_recv_status_on_client_error_string(this));
```

Comment:
naming consistency nit: suggest calling this: `debugErrorString`

Reply:
done.